README.md

```diff
@@ -1 +1,156 @@
-# opencraft
+OpenCraft
+=========
+
+Install
+-------
+
+Instructions based on Ubuntu 14.04.
+
+Install the system package dependencies & virtualenv:
+
+```
+$ sudo apt-get install `cat debian_packages.lst`
+$ pip3 install --user virtualenv && pip3 install --user virtualenvwrapper
+```
+
+Ensure you load virtualenv with Python 3 in `~/.bashrc`:
+
+```
+export PATH="$PATH:$HOME/.local/bin" VIRTUALENVWRAPPER_PYTHON=/usr/bin/python3
+source $HOME/.local/bin/virtualenvwrapper.sh
+```
+
+Then reload `~/.bashrc`, create the virtual env and install the Python requirements:
+
+```
+$ . ~/.bashrc
+$ mkvirtualenv -p /usr/bin/python3 opencraft
+$ pip install -r requirements.txt
+```
+
+
+Configure
+---------
+
+Create an `.env` file at the root of the repository or set environment variables, customizing the
+settings from `opencraft/settings.py` which are loaded via `env()`.
```

Comment:
@bradenmacdonald It would be nice, but to give a working `.env` file as an example in the README would require publishing auth credentials, like OpenStack cluster env vars or Gandi API token keys. Without these, you wouldn't really be able to do much - the closest there is to a working `.env` file without real auth credentials is the `.env.test` file, but it only works because tests mock out all the external API calls. Imho for someone setting this up, besides getting such an example from someone who would trust you with these auth tokens, there isn't much of a way around reading the `opencraft/settings.py` file, and going to create accounts on these services to gather the credentials. That's why I included lengthy descriptions of the options in that file, to make this a guided process.